src/utils/actions.js

```diff
@@ -275,7 +275,13 @@ export const postFile = (
         if(!isObjectEmpty(fileMetadata)) {
             Object.keys(fileMetadata).forEach(function (key) {
                 let value = fileMetadata[key];
-                req.field(key, value);
+                if (Array.isArray(value)) {
```

Comment:
@tomrndom do not duplicate code
just create a helper method so knowledge its on one single place
ie: normalizeFormDataPayload